src/steps/05-PerformanceRequirements/DOW/ServiceOfferingDetails.vue

```diff
@@ -129,50 +198,57 @@ export default class ServiceOfferingDetails extends Vue {
     return getIdText(string);
   }
 
-  public mounted(): void {
-    // get this from store data when implemented 
-    this.categoryName = "Data Management";
+  @Watch("selectedOptions")
+  public selectedOptionsChange(newVal: string[]): void {
+    this.isIL6Selected
+      = newVal.indexOf('405b52af87970590ec3b777acebb3556') > -1 ?
```

Comment:
Use the `impact_level` to determine if this is IL6. the `sys_id` could change on different envs. Either need to determine this by looking into the classifications array, or by the label in the checkbox array.